src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -430,6 +430,8 @@ public static function create(string $uri, string $method = 'GET', array $parame
      * This is mainly useful when you need to override the Request class
      * to keep BC with an existing system. It should not be used for any
      * other purpose.
+     *
+     * @param callable():SessionInterface|null $callable A PHP callable
```

Comment:
```suggestion
     * @param callable():SessionInterface|null $callable
```

I like the clear parameter, but the "A PHP callable" did not bring any added value.